Source/WebCore/platform/mediarecorder/MediaRecorderPrivateGStreamer.cpp

```diff
@@ -389,7 +388,7 @@ GRefPtr<GstEncodingContainerProfile> MediaRecorderPrivateBackend::containerProfi
             gst_encoding_profile_set_restriction(m_audioEncodingProfile.get(), restrictionCaps.leakRef());
         }
 
-        gst_encoding_container_profile_add_profile(profile.get(), m_audioEncodingProfile.get());
+        gst_encoding_container_profile_add_profile(profile.get(), GST_ENCODING_PROFILE(gst_object_ref(m_audioEncodingProfile.get())));
```

Comment:
I wonder if we should create a method in GRefPtr to return a transfer full reference. It would be nice. Then you could avoid the _ref and the cast.